src/java/com/puppetlabs/ssl_utils/ExtensionsUtils.java

```diff
@@ -536,7 +546,7 @@ private static Extensions getExtensionsFromCRL(X509CRL crl)
      *         provided OID.
      * @throws IOException
      */
-    private static ASN1Object binaryToASN1Object(ASN1ObjectIdentifier oid,
+    public static ASN1Object binaryToASN1Object(ASN1ObjectIdentifier oid,
```

Comment:
why is this public?

Reply:
oh, this got changed when I was experimenting with ways to generate new AuthorityKeyIds, I'll revert it back.